samples/attested_tls/server/host/host.cpp

```diff
@@ -43,15 +43,30 @@ int main(int argc, const char* argv[])
     oe_result_t result = OE_OK;
     int ret = 1;
     char* server_port = NULL;
+    bool keep_server_up = false;
 
     /* Check argument count */
     if (argc != 3)
     {
+        if (argc == 4)
+        {
+            char* loop_option = (char*)"-server-in-loop";
+            size_t loop_option_len = 0;
+            loop_option_len = strlen(loop_option);
+            if (strncmp(argv[3], loop_option, loop_option_len) != 0) {
```

Comment:
why strncmp and not strcmp?  Won't this match things like -server-in-loophole?  I'm not sure that's desirable.
```suggestion
            if (strcmp(argv[3], loop_option) != 0) {
```

Reply:
Thanks for catching this bug!